pkg/aggregate/iterator.go

```diff
@@ -116,7 +116,8 @@ func (as *AggregateSeries) NewSetFromAttrs(
 	}
 
 	for _, aggr := range rawAggregators {
-		if aggr&as.aggrMask != 0 {
+		// Always have count aggregator by default
+		if aggr&as.aggrMask != 0 || aggr&aggrTypeCount != 0 {
```

Comment:
aggr&aggrTypeCount != 0 will always be TRUE

Reply:
Yes, it will always be TRUE once per loop. that way the count aggregator will be by default always as said in the comment above.

Reply:
@talIguaz i think its better to just set aggrMask to include count (i.e. |= aggrTypeCount) in NewAggregateSeries assuming aggrMask!= 0  (line 54)